docs/content/get-started/set-up.md

```diff
@@ -63,19 +63,19 @@ The SpatialOS GDK for Unity source code is hosted on GitHub. You need to downloa
 
  **Using Visual Studio?**
 
-As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find this by clicking on the **More** option for *Visual Studio Build Tools* in the Visual Studio Installer, and selecting **Modify** from the drop-down menu.
+As you install [Visual Studio](https://www.visualstudio.com/downloads/), click on the **Workloads** tab in the Installer. If you already have Visual Studio installed, you can find this by clicking the **Modify** option for Visual Studio Community 2017. If the **Modify** option is not visible, this option may also be found in the drop-down menu that appears when you select the **More** option.
```

Comment:
doesn't have to be the community version :)

also: I have never seen that modify option, I wonder which option we should show first as I suspect that most people will not read the last sentence ;)

Reply:
![capture](https://user-images.githubusercontent.com/19690292/52212823-539bed80-2885-11e9-8cda-af15cbed6d16.PNG)

Good shout. I've removed "Community", tweaked some language and added bullet-points in attempt to give this section more of an obvious though line / call to action.